addTourpackageProcess.php

```diff
@@ -0,0 +1,167 @@
+<?php
+require 'connection.php';
+
+$jsonText = $_POST["tourPackageData"];
+$dataObject = json_decode($jsonText);
+
+$name = $dataObject->name;
+$price = $dataObject->price;
+$description = $dataObject->description;
+$destinationList = explode(",", $dataObject->destinationList);
+$noOfVehicles = $dataObject->nuOfVehicles;
```

Comment:
This data is useless, because you already have `vehicleList` array. So, you can simply get the length of the array as number of vehicles.

Reply:
Here, the way you recommend does not provide the output that I want finally. Because, the values I want to insert to the database, are get from a text included in a span tag. If I use the way you are talking, it divides the each character in the text and push into an array. That's not I want. For example, I want to insert the values in this array - [12, 21, 35, 37] and let's imaging this array is included in a span tag. If I tried to get the first element of this, it will output '1' as the first element. That's not what I want. I want '12' as the first element. This is because, we cannot take the value in a span tag using 'document.getElementById("id").value' method. Here I have used '.innerText' instead of '.value'. Therefore, It provides the value inside the span tag as a text even arrays. Therefore, here I have used the explode method to divide text from each ',' symbol.